docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -270,7 +270,7 @@ docker run --rm \
 	-v /path/to/data:/data:rw \
 	--name flow-go \
 	--network host \
-	gcr.io/flow-container-registry/access:v0.18.4 \
+	gcr.io/flow-container-registry/access:v0.19.1 \
```

Comment:
@zhangchiqing - what will the tag be to tomorrow?

Reply:
Let's use v0.19.2, it includes the fix for synchronization engine and default flag changes.